.github/workflows/integrity-check.yml

```
@@ -11,69 +11,82 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
+env:
+  CI: true
+  PNPM_CACHE_FOLDER: .pnpm-store
+
 jobs:
   security-audit:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout source
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
 
+      - name: install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
       - name: Set up Node.js
         uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
         with:
           node-version: 20
 
       - name: Report known vulnerabilities
-        run: npm audit
+        run: pnpm audit
 
   test-with-node:
     runs-on: macos-latest
     steps:
       - name: Checkout source
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
 
+      - name: install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
       - name: Set up Node.js
         uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
         with:
           node-version: 20
           registry-url: https://registry.npmjs.org/
 
-      - name: Install latest npm
-        run: npm install -g npm@latest
-
       - name: Install dependencies
-        run: npm ci
+        run: pnpm install
 
       - name: Build all workspace packages
-        run: npm run build
+        run: pnpm run build
```

If I'm understanding the docs right, there is no `pnpm run` (tho perhaps an alias exists) so instead this is just... 

```suggestion
        run: pnpm build
```

---

okay yeah getting my hands dirty it must support an alias, but I would recommend we go with [the official docs](https://pnpm.io/pnpm-cli#commands) and remove the unnecessary `run` command

---

Done in [commit below](https://github.com/TBD54566975/tbdex-js/pull/30/commits/02e0e357aa39c285758d65485f95b6d83ec68c41)